mrp_workorder_cancel/views/mrp_workorder_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="mrp_production_workorder_cancel_tree_view_inherit" model="ir.ui.view">
+        <field name="name">mrp_production_workorder_cancel_tree_view_inherit</field>
+        <field name="model">mrp.workorder</field>
+        <field
+            name="inherit_id"
+            ref="mrp.mrp_production_workorder_tree_editable_view"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="decoration-muted">state=='cancel'</attribute>
+            </xpath>
+            <button name="button_start" position="after">
+                <button
+                    name="button_cancel"
```

Comment:
This should call action_cancel and have `confirm="True"` to popup the confirmation on click.
The wizard is not really needed.

Reply:
Hello @dreispt 
When we add Button in List View Odoo List View dont support confirm Attribute.
So we need to add Wizard.